masonry/src/passes/update.rs

```diff
@@ -584,6 +584,17 @@ pub(crate) fn run_update_pointer_pass(root: &mut RenderRoot) {
 
     let pointer_pos = root.last_mouse_pos.map(|pos| (pos.x, pos.y).into());
 
+    if root.global_state.inspector_state.is_picking_widget {
+        if let Some(pos) = pointer_pos {
+            root.global_state.inspector_state.hovered_widget = root
+                .get_root_widget()
+                .find_widget_at_pos(pos)
```

Comment:
This doesn't seem to find the label inside a `Button`, for some reason

Reply:
The search uses pointer routing, which labels inside buttons opt out of. We should ideally fix that, but it requires more design work.